2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
Sorry, I was suggesting always an array.
Then you could easily have any number of formats (if more than 2 are ever needed) and wouldn't rely on them being in a pattern like above:
`if (format === 'currency,thousands')`

And you wouldn't have to split that string to get there.